org.bndtools.templates.osgi/resources/templates/ds/bnd.bnd

```diff
@@ -1,15 +1,9 @@
 -buildpath: \
-	osgi.annotation; version=6.0.1,\
-	osgi.core; version=6.0,\
-	osgi.cmpn; version=6.0
+	osgi.annotation; version=latest,\
+	osgi.core; version=latest,\
+	osgi.cmpn; version=latest
```

Comment:
I think we should only include osgi.core.

The osgi.cmpn was really a bad idea. It was useful at the time but with the smaller API jars we have a much better model

Reply:
Adressed